xbmc/guilib/GUIFontCache.cpp

```diff
@@ -23,46 +23,66 @@
 #include "GUIFontTTF.h"
 #include "GraphicContext.h"
 
-#if defined(TARGET_DARWIN )
-// apple clang with c++11 doesn't like
-// the forwards declarations of boost
-// disable them ...
-#define BOOST_DETAIL_NO_CONTAINER_FWD 1
-#endif
-
-#include "boost/multi_index_container.hpp"
-#include "boost/multi_index/sequenced_index.hpp"
-#include "boost/multi_index/hashed_index.hpp"
-#include "boost/multi_index/member.hpp"
-
-using namespace boost::multi_index;
-
 template<class Position, class Value>
 class CGUIFontCacheImpl
 {
-  /* Empty structs used as tags to identify indexes */
-  struct Age {};
-  struct Hash {};
-
-  typedef multi_index_container <
-    CGUIFontCacheEntry<Position, Value>,
-    indexed_by<
-    sequenced<tag<Age> >,
-    hashed_unique<tag<Hash>,
-    member<CGUIFontCacheEntry<Position, Value>,
-    CGUIFontCacheKey<Position>, &CGUIFontCacheEntry<Position, Value>::m_key
-    >,
-    CGUIFontCacheHash<Position>, CGUIFontCacheKeysMatch < Position >
-    >
-    >
-  > EntryList;
-
-  typedef typename EntryList::template index<Age>::type::iterator EntryAgeIterator;
-  typedef typename EntryList::template index<Hash>::type::iterator EntryHashIterator;
+  struct EntryList
+  {
+    typedef std::multimap<size_t, CGUIFontCacheEntry<Position, Value>*> HashMap;
```

Comment:
could we get rid of all the typename stuff by switching these to `using HashMap = std::multimap<size_t, CGUIFontCacheEntry<Position, Value>*>;`? I'm a bit rusty on the rules regarding this as VS doesn't follow the standard in that regard.